titiler/stacapi/templates/wmts-getcapabilities_1.0.0.xml

```diff
@@ -113,7 +113,7 @@
   {% for tms in tilematrixsets %}
   <TileMatrixSet>
     <ows:Identifier>{{ tms.id }}</ows:Identifier>
-    <ows:SupportedCRS>{{ tms.crs.srs }}</ows:SupportedCRS>
+    <ows:SupportedCRS>urn:ogc:def:crs:{{ tms.id }}</ows:SupportedCRS>
```

Comment:
this is not valid, a TMS `identifier` is not always a valid SRS/CRS string 

what we could do is use `tms.crs.to_epsg()` 
```
{% if tms.crs.to_epsg() %}
<ows:SupportedCRS>urn:ogc:def:crs:epsg:{{tms.crs.to_epsg()}}</ows:SupportedCRS>
{% else %}
<ows:SupportedCRS>{{ tms.crs.srs.replace("http://www.opengis.net/def/", "urn:ogc:def:").replace("/", ":")}} </ows:SupportedCRS>
{% endif %}
```

Reply:
we don't have tests for the WMTS endpoint so could you tests those change before we merge?